### DISCUSSION TITLE
SMTP Help

### DISCUSSION THREAD

**Discussion by apixler77** [utmstack/UTMStack#609] [General]
Cannot for the life of me get the notifications SMTP setup. I am using Gmail SMTP with an app password. Anyone have any pointers?

**Comment by c3s4rfred** [comment]
Hi, @apixler77 can you tell us what options you have selected in the settings configuration and if your SMTP server has SSL or STARTTLS. Try to send us some application log about email error. You can find logs in the backend docker container and inside the web application -> App Management ![image](https://github.com/utmstack/UTMStack/assets/17603499/201df4a0-6768-4dc9-8ea5-29be70778f8f) -> Settings -> Application Logs

Best regards

**Reply by apixler77** [reply]
@c3s4rfred 

Thanks so much for your reply, I think I was fat fingering the PW when adding it. I have it all configured and working now. Thank you again.
